### DISCUSSION TITLE
Configure LME to only show certain servers

### DISCUSSION THREAD

**Discussion by Gschragg** [cisagov/LME#178] [FAQ] [answered]
I have someone in our organization that we want to allow to see the LME on elastic but only for the servers they control. I wasn't sure if there was anyone who a premade dashboard for that or how I would go bout setting that up. I'm aware you can setup individuals users on elastic to look at the data, but wanted more restrictions to what hosts they can see.

**Comment by llwaterhouse** [comment, marked as answer]
Yes you can do that by creating elasticsearch security roles and permissions.  The level of security roles and permissions depends on the level of Elastic you have.
